google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh

```diff
@@ -46,6 +46,43 @@ go test -v \
   -skip "Generic_CloseClient|Generic_DeadlineExceeded|NoRetry_OutOfOrderError_Reverse|Retry_LastScannedRow_Reverse|Retry_WithRetryInfo_OverallDedaline|TestExecuteQuery" \
   -proxy_addr=:9999
 exit_status=$?
+
+# Run all the ExecuteQuery tests that either work or we plan to skip such as
+# CloseClient
+go test -v \
+  -run "TestExecuteQuery" \
+  -skip "CloseClient|FailsOnEmptyMetadata|FailsOnExecuteQueryMetadata|FailsOnInvalidType|FailsOnNotEnoughData|FailsOnNotEnoughDataWithCompleteRows|FailsOnSuccesfulStreamWithNoToken|ChecksumMismatch|FailsOnTypeMismatch|FailsOnTypeMismatchWithinMap|FailsOnTypeMismatchWithinArray|FailsOnTypeMismatchWithinStruct|FailsOnStructMissingField|RetryTest_WithPlanRefresh|PlanRefresh" \
+  -proxy_addr=:9999
+exit_status=$?
+
+# These next four go test commands group the currently failing ExecuteQuery
+# tests into groups that exercise similar functionality and should be worked on
+# together.
+
+# Metadata tests b/461232934
+#go test -v \
+#  -run "FailsOnEmptyMetadata|FailsOnExecuteQueryMetadata|FailsOnInvalidType" \
+#  -proxy_addr=:9999
+#exit_status=$?
```

Comment:
To make sure that I'm understanding this correctly, if the `FailsOnEmptyMetadata` test is failing then does it mean that the client is not returning the expected INTERNAL error with the "Invalid empty ResultSetMetadata received" message as shown in https://github.com/googleapis/cloud-bigtable-clients-test/blob/f046dcf7b194550ad61a95419f8b198dce7fdd6c/tests/executequery_test.go#L710C4-L710C44?